test/layers_insert.test.js

```diff
@@ -0,0 +1,32 @@
+/* eslint no-shadow:[0] */
+import test from "tape";
+import MapboxDraw from '../';
+import createMap from "./utils/create_map";
+import spy from "sinon/lib/sinon/spy";
+
+test("Layers insert test", (t) => {
```

Comment:
It would be beneficial to test that the drawing functionality (i.e. adding + displaying vertices) still works as expected when drawing "beneath" a layer specified with `insertBeforeLayerId` in a region where said layer has features defined. That is, suppose `insertBeforeLayerId` is set to `'layer'`, and `'layer'` has a polygon feature covering a geographic region where the user is drawing.

Adding a test to clarify the intended behavior of selecting a drawn layer being "covered" by a feature in `'layer'` would also be helpful.